WcaOnRails/app/views/competitions/_competition_form.html.erb

```diff
@@ -265,6 +266,11 @@
     <hr>
     <%= f.input :remarks, disabled: is_actually_confirmed %>
 
+    <% if @competition.new_record? && @competition.cloned_from && !@competition.cloned_from.competition_tabs.empty? %>
```

Comment:
`clone_tabs` is the attribute shown as check box to allow the user to make decision whether he wants to clone tabs or not, and thus I don't think it can be used as you suggested. But it gave me an idea of using `<% unless @competition.clone_tabs.nil? %>`.
`clone_tabs` is set to true after we garb a competition from `build_clone` and then user editing the form passes either true of false via check box, so it seems to be a boolean when we are cloning and nil otherwise.  What do you think of it?